tests/snapshots/files__branch_hoisting-optimize.snap

```diff
@@ -4,45 +4,32 @@ expression: visualization.result
 ---
 # ARGS: 0
 @main(v0: int) {
```

Comment:
This one looks suspicious. It seems the old one does loop unrolling and the new one does not.